### PR TITLE
docs(S3): update mentions of S3 in documentation MTA-5188

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: storage/object/api-cli/installing-rclone.mdx [hunk 1]
```diff
@@ -14,7 +14,7 @@ categories:
   - object-storage
 ---
 
-[Rclone](https://rclone.org) is a command-line tool that can be used to manage your cloud storage. It communicates with any S3-compatible cloud storage provider as well as other storage platforms.
+[Rclone](https://rclone.org) is a command-line tool that can be used to manage your cloud storage. It communicates with any Amazon Amazon S3-compatible cloud storage provider as well as other storage platforms.
```
Comment (RoRoJ) [inline storage/object/api-cli/installing-rclone.mdx:17]:
```suggestion
[Rclone](https://rclone.org) is a command-line tool that can be used to manage your cloud storage. It communicates with any Amazon S3-compatible cloud storage provider as well as other storage platforms.
```

File: storage/object/api-cli/managing-lifecycle-cliv2.mdx [hunk 2]
```diff
@@ -98,7 +98,7 @@ categories:
   ```
 </Message>
 
-## Installing a configuration file for S3 tools (s3cmd, rclone, and mc)
+## Installing a configuration file for Amazon Amazon S3-compatible tools (s3cmd, rclone, and mc)
```
Comment (RoRoJ) [inline storage/object/api-cli/managing-lifecycle-cliv2.mdx:101]:
```suggestion
## Installing a configuration file for Amazon S3-compatible tools (s3cmd, rclone, and mc)
```

File: storage/object/index.mdx [hunk 3]
```diff
@@ -7,7 +7,7 @@ meta:
 <ProductHeader 
    productName="Object Storage"
    productLogo="objectStorage"
-   description="Object Storage allows you to store and access data from the Scaleway console and S3-compatible clients."
+   description="Object Storage allows you to store and access data from the Scaleway console and Amazon Amazon S3-compatible clients."
```
Comment (RoRoJ) [inline storage/object/index.mdx:10]:
```suggestion
   description="Object Storage allows you to store and access data from the Scaleway console and Amazon S3-compatible clients."
```

File: storage/object/reference-content/s3-iam-permissions-equivalence.mdx [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
-  description: Understand how IAM permissions in S3 relate to Scaleway Object Storage.
+  title: Amazon S3 and IAM permissions equivalence
+  description: Understand how IAM permissions in Amazon S3 relate to Scaleway Object Storage.
 content:
-  h1: S3 and IAM permissions equivalence
-  paragraph: Understand how IAM permissions in S3 relate to Scaleway Object Storage.
-tags: object-storage s3 aws action equivalent iam permission set
+  h1: Amazon S3 and IAM permissions equivalence
+  paragraph: Understand how IAM permissions in Amazon S3 relate to Scaleway Object Storage.
+tags: object-storage Amazon-S3 aws action equivalent iam permission set
```
Comment (RoRoJ) [inline storage/object/reference-content/s3-iam-permissions-equivalence.mdx:8]: are tags always meant to be lower case? Not sure

**Review by RoRoJ — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**